docs/how-to/configure.md

```diff
@@ -178,26 +178,30 @@ A brief message will confirm the creation of your new account and you may now lo
 !!! alert alert-info "Note"
     This tab appears only when *Advanced Settings* --> *Integrated Alerting* is on.
 
-Alertmanager communications settings.
-
+Global communications settings for [Integrated Alerting](../using/alerting.md).
 
 ![](../_images/PMM_Settings_Communication.jpg)
 
+Integrated Alerting uses a separate instance of Alertmanager. The descriptions for the settings here are reproduced from [Prometheus Alertmanager configuration](https://prometheus.io/docs/alerting/latest/configuration/).
```

Comment:
> Integrated Alerting uses a separate instance of Alertmanager

let's be a bit verbose here. The separate alertmanager used for integrated alerting is already run by pmm-managed so we should add the information here so that the user doesn't get confused and think about starting their own separate instance of alertmanager to use Integrated Alerting.

"Integrated Alerting uses a separate instance of Alertmanager run by pmm-managed or the pmm server (whichever term is used in the docs)"